src/plays/pokemon-stats/typing.d.ts

```diff
@@ -0,0 +1,36 @@
+export interface Species {
```

Comment:
@yung-coder , Can we change the file name from `typing.d.ts` to `types.d.ts`?

Reply:
okay